### DISCUSSION TITLE
PWRMON v1.21 - A Tesla Powerwall Monitoring Script for Asus Routers

### DISCUSSION THREAD

**Discussion by ViktorJp** [vloschiavo/powerwall2#85] [Show and tell]
Thanks to this excellent github post/content, I was able to build a powerwall monitoring tool in POSIX shell script, meant to be run from an Asus Router.  Recreated the Tesla powerwall app powerflows, including dashboards showing solar, home, grid and battery usage. Also created a large dashboard for use of displaying in your home to see current stats from across the room.

Link to download/discussions:
https://www.snbforums.com/threads/pwrmon-v1-21-oct-3-2022-monitor-your-tesla-powerwall-s-solar-grid-home-stats-new-weather-sms-email-notifications-large-dashboard.80203/

Github Project:
https://github.com/ViktorJp/PWRMON

Screenshots:
[Page 1](https://imgur.com/a/1NZfq5I)
[Page 2](https://imgur.com/a/CEMcjDL)
[Page 3](https://imgur.com/a/yfMVDoR)

**Comment by vloschiavo** [comment]
Super cool!  Are you ok if I link to your project here?

**Reply by ViktorJp** [reply]
Please feel free to!  Hoping there's some good examples in there that others can make use of as well.  ;)
